pages/common/kube-capacity.md

```diff
@@ -0,0 +1,16 @@
+# kube-capacity
+
+> This is a simple CLI that provides an overview of the resource requests, limits, and utilization in a Kubernetes cluster.
+> It attempts to combine the best parts of the output from kubectl top and kubectl describe into an easy to use CLI focused on cluster resources.
```

Comment:
Perhaps this will read better?
```suggestion
> Combine the best parts of `kubectl top` and `kubectl describe` into a CLI focused on cluster resources.
> Homepage: <https://github.com/robscott/kube-capacity>.
```

We mention it's a "simple CLI" in the above line, so I'm not sure we need "easy to use" in this one.